nusantara/nusa_datasets/nergrit/nergrit.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" NERGrit Dataset """
+
+from pathlib import Path
+from typing import List
+
+import datasets
+
+from nusantara.utils import schemas
+from nusantara.utils.common_parser import load_conll_data
+from nusantara.utils.configs import NusantaraConfig
+from nusantara.utils.constants import Tasks
+
+_CITATION = """\
```

Comment:
Not sure about this but since this dataset is part of IndoNLU split should we just cite IndoNLU paper here? 
For consistency with our nusa catalogue too https://indonlp.github.io/nusa-catalogue/card.html?nergrit

cc: @SamuelCahyawijaya @holylovenia

Reply:
I think we should cite both the NERGrit and IndoNLU paper here. Thank you @christianwbsn for mentioning this

Reply:
I added both to the citation list as suggested.